visyn_core/__init__.py

```diff
@@ -66,3 +66,19 @@ def register(self, registry: RegHelper):
             "visyn_core.security.store.oauth2_security_store",
             {},
         )
+
+    def paths_without_authentication(self):
+        from . import manager
+
+        return (
+            "/api/sentry",
+            "/api/login",
+            "/api/logout",
+            "/api/security/stores",
+            "/api/v1/visyn/clientConfig",
+            "/api/health",
+            "/api/metrics",
+            "/health",
+            "/metrics",
```

Comment:
```suggestion
            "/api/health",
            "/api/login",
            "/api/logout",
            "/api/metrics",
            "/api/security/stores",
            "/api/sentry",
            "/api/v1/visyn/clientConfig",
            "/health",
            "/metrics",
```

Sort the list alphabetically for easier lookup.